core/templates/dev/head/pages/exploration_editor/simple_editor_tab/simple_editor_sidebar_directive.html

```diff
@@ -12,13 +16,16 @@
 
     <li class="sidebar-item" ng-repeat="question in questionList.getQuestions() track by $index">
       <div id="<[ID_PREFIX + question.getId()]>" class="sidebar-item-header">
-        <i ng-click="question.toggleCollapse()" class="collapse-icon material-icons">
+        <i ng-click="question.toggleCollapse()" class="collapse-icon material-icons" ng-hide="SimpleEditorSidebarModeService.isSidebarInEditMode()">
           <[isCollapsed(question) ? '&#xE5C7;' : '&#xE5C5;']>
         </i>
+        <i ng-click="deleteQuestion(question)" class="delete-icon material-icons" ng-hide="SimpleEditorSidebarModeService.isSidebarInReadonlyMode()">
+          <['&#xE872;']>
+        </i>
         <strong ng-click="scrollToQuestion(question)">Question <[$index + 1]></strong>
       </div>
 
-      <ul ng-show="question.isCollapsed()">
+      <ul ng-show="question.isCollapsed() && (SimpleEditorSidebarModeService.isSidebarInReadonlyMode())">
```

Comment:
There shouldn't be a need to put parenthesis for `SimpleEditorSidebarModeService.isSidebarInReadonlyMode()`, what I'd recommend is create a new variable and use that smaller variable name.